.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+# Unix-style newlines with a newline ending every file
```

Comment:
IMO this comment is still kinda redundant because it's explaining *what* the code already does ;)